public/store/css/main.css

```diff
@@ -829,7 +829,7 @@ HISTORY PAGE
 	text-align: center;
 }
 
-.history  > div > p {
+.history > p {
```

Comment:
Should it be #d2232a, rather than red?

Not going to worry about it now at all, but I'm not a huge fan of child selectors from page-level classes. It can make it difficult to change things later.

Reply:
👍 you right, fixed for this instance